### ISSUE TITLE
Keep the same position when the image is changed.

### ISSUE THREAD

**Issue by n0aaz** [vgilabert94/streamlit-image-zoom#8]
I have an issue that makes comparing images difficult with this tool : 
Whenever the image is changed (through a st.radio for example) there is no way to tell the component to keep the same zoomed area, even in dragclick mode. Thus when you want to see a before/after , the zoom is reset every time.

Is there a way to add a mode that would keep the zoom between image changes?

This issue can be seen in the online demo too : 
https://image-zoom.streamlit.app/

If you zoom in dragclick mode and then change the zoom factor or 'keep_resolution' , the zoom gets reset.

**Comment by vgilabert94** [comment]
Yes, I like your proposal. Could you do some PR for this feature?
